Community/README.md

```diff
@@ -0,0 +1,48 @@
+# Community
+
+Entrypoint for getting onboarded.
+
+## Tools used by the community
+
+- Instant messaging
+  - [Matrix](https://matrix.to/#/!TiDqlLmEUaXqTemaLc:matrix.org?via=matrix.org)
+  - [Nextcloud Talk](https://scs.sovereignit.de/nextcloud) (Not used)
+  - [Fosstodon](https://fosstodon.org/@sovereigncloudstack) (Only one-directional)
+- Annoucements
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+  - [RSS Feed](https://scs.community/feed.xml)
+  - [Fosstodon](https://fosstodon.org/@sovereigncloudstack)
+  - [Twitter](https://twitter.com/scs_osballiance)
+  - [LinkedIn](https://www.linkedin.com/showcase/sovereigncloudstack)
+  - [Website](https://scs.community)
+- Video-Calls
+  - [Jitsi](https://conf.scs.koeln:8443/SCS-TECH)
+- Groupware/Fileshare/Organizational Stuff
+  - [Nextcloud](https://scs.sovereignit.de/nextcloud)
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+- Task distribution
+  - [Wekan](https://scs.sovereignit.de/wekan)
+  - [GitHub](https://github.com/SovereignCloudStack)
+  - [Mailing-List](https://scs.sovereignit.de/mailman3/postorius/lists/announce.lists.scs.community/)
+- Guides
+  - [GitHub](https://github.com/SovereignCloudStack)
+  - [Contributor Guide](https://scs.community/docs/contributor/)
+
+### Proposal to improve (Just my two cents)
+
+- reduce usage of mailing lists drastically
+- reduce amount of mailing lists
+- use single source of annoucement
+- don't spam the mailing lists with complete annoucements, but rather the link to the annoucement!
+- ditch Wekan and move to GitHub to reduce tool clutter
+- provide ONE single URL for all services with proper subdomains:
+  - nextcloud.sovereignit.de
+  - lists.sovereignit.de
+  - jitsi.sovereignit.de
+  - sovereignit.de (website)
+  - I've never understood why there is scs.community and scs.sovereignit.de
```

Comment:
sovereignit.cloud is also in use for the gx-scs demonstrator.

maybe something like "debian.org vs. debian.net"... sorry I can't find the page where it pointed you to when to use what domain.

https://wiki.debian.org/MemberBenefits